frontend/src/components/Error/Error.jsx

```diff
@@ -0,0 +1,24 @@
+import "./Error.css";
+import { useState, useEffect, useContext } from "react";
+import { UserContext } from "../App/App";
+
+export default function Error({ error }) {
+    const [errorMessage, setErrorMessage] = useState(error);
```

Comment:
I don't think we need a local state, accessing the global state error should suffice. 
Keeping track of two pieces of state for the same thing makes things more complex.